toolbox/tabs/grids/GroupedGridPanel.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import {hoistComponent} from 'hoist/core';
+
+@hoistComponent()
+export class GroupedGridPanel extends Component {
```

Comment:
Great - this will live alongside the standard grid (no grouping) and soon a treeGrid example.